export default function catchError(err) {

}
